packages/core/src/components/TermList/Term.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import withStyles, { WithStylesProps } from '../../composers/withStyles';
+
+import Row from '../Row';
+import Spacing from '../Spacing';
+import Text from '../Text';
+
+function Term({
+  label,
+  after,
+  endAlign,
+  keyTextRegular,
+  children,
+  cx,
+  styles,
+}: {
+  label: string;
+  after?: NonNullable<React.ReactNode>;
+  endAlign?: boolean;
+  inline?: boolean;
+  keyTextRegular?: boolean;
+  children: NonNullable<React.ReactNode>;
+} & WithStylesProps) {
+  return (
+    <div className={cx(styles.container)}>
+      <Row inline={!endAlign} after={endAlign && after}>
```

Comment:
i would move this all into the `dt`, so the `div` isn't necessary. in a dl - it's expected children are dt/dd only

```jsx
<>
  <dt>
    <Row inline={!endAlign} after={after}>
      <Text small={small}>{label}</Text>
    </Row>
  </dt>

  <dd className={cx(styles.dd)}>
    <Text>{children}</Text>
  </dd>
</>
```

and some cleanup around endAlign/inline/after:
- if endAlign is true, `after` goes to the end by default, so don't  need to check
- Row's `inline` makes its children inline/close, but not the entire thing (its still block level), so don't need to to the extra `!endAlign && <Spacing>` after the dt.

Reply:
Not 100% sure I followed the last point, but here's what I ended up with
```
 <>
      <dt>
        <Row after={endAlign && after}>
          <Text inline small={!keyTextRegular}>
            {label}
          </Text>
          {!endAlign && (
            <Spacing inline left={1}>
              {after!}
            </Spacing>
          )}
        </Row>
      </dt>
      <dd className={cx(styles.value)}>
        <Text>{children}</Text>
      </dd>
    </>
```